Gemfile.lock

```diff
@@ -476,6 +480,7 @@ GEM
       pry (>= 0.13, < 0.15)
     pry-rails (0.3.9)
       pry (>= 0.10.4)
+    psych (3.3.4)
```

Comment:
I'm not sure if locking the version of Psych is a productive option. Might we want to seek for modifying the code to be able to run on older and newer versions of Psych?

Reply:
Would require I belive a Rails 7 upgrade

Reply:
[rails-settings-cached-2.8.2 is compatible with Rails 5.0.0 and up](https://github.com/huacnlee/rails-settings-cached/blob/v2.8.2/rails-settings-cached.gemspec#L28) and I guess we'd need to someday be able to use Psych >= 4 for safer code base.

On the other hand, it seems [rails-settings-cached-2 introduced a number of breaking changes](https://github.com/huacnlee/rails-settings-cached/releases/tag/v2.0.0) and there seems to be a long way for us to migrate from `config/settings.yml` to `app/models/setting.rb` and retain scoped settings.

Might there be expert who has more insight into this?

Reply:
Upgrading [rails-settings-cached-2](https://github.com/huacnlee/rails-settings-cached/releases/tag/v2.0.0) would help, but I think i also saw a failure in Rails 6